optional_requirements.txt

```
@@ -3,4 +3,6 @@ aiohttp==3.7.4.post0
 slackclient==2.7.2
 responses==0.10.14
 google-cloud-translate==2.0.1
-paramiko>=2.8.0
\ No newline at end of file
+paramiko>=2.8.0
+boto3>=1.21.21
+google>=3.0.0
```

I think you have the wrong package imported here.  It looks like `google` is related to the search engine, while `google-cloud-storage` is the installation name for the client that includes cloud storage.  Confusingly, you `import google` with either of them.
```suggestion
google-cloud-storage>=2.4.0
```